Contributing/How-to-Contribute-Code.md

```diff
@@ -0,0 +1,35 @@
+## Code Contributions
+
+We would like to thank you for your interest in contributing. You can find below how you can get started with knowing and contributing to keploy. The first few steps are always crucial but signify how you wish to involve with the community.
+
+1.  Get to know keploy:
+
+    -   Read the [Contributing guidelines](https://github.com/keploy/template/blob/main/CONTRIBUTING.md)
+    -   Read the [Code of Conduct](../CODE_OF_CONDUCT.md)
+    -   Review the [Coding Guidelines]() and the [Contribution Criteria](../CONTRIBUTING.md).
+    -   Review the bug-list, open issues, Project Board (if you want to code a new feature) or the [Plan](). To start a new Project follow these [guidelines](How-to-Start-a-New-Project.md).)
+    -   Contact the Maintainers, if available in the README document, or the Repository Maintainer by submitting an [Issue]() with an @mention.
+
+2.  [Join the keploy organisation]()  
+
+3.  [Fork the repository](https://help.github.com/en/articles/fork-a-repo) if you do not want to join keploy or want to start working before your request to join has been approved.
+
+4.  Make your contribution
+
+    -  Make changes or additions to the code.
+    -  Ensure that your code meets the [coding style guidelines]().
+    -  Perform relevant tests, which includes unit-testing or end-to-end testing as found fit for the repository that you will be submitting the pull-request for. 
+    -  Follow the [documentation guidelines]().
+
+5.  Submit your contribution
+
+    -   [Submit a Pull-Request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to the Branch you want to contribute.
+    -   Use a clear and descriptive title.
+    -   Ensure maintainers can understand your proposed changes from the description.
+    -   Ensure your Pull-Request passes all checks and has no conflicts.
+    -   Click `Create Pull Request`.
+    -   Include in the comments section the tests that you have completed and if they were a pass or fail.
+
+6.  Get credit for your work
+
+    -   [Claim credit]() if you are not listed yet.
```

Comment:
Why is there an empty link?

Reply:
Because many folders and many things are not done